atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -3,8 +3,8 @@ scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
 default_scan: redhat-release
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
```

Comment:
I agree with @dharmit we should have separate names for all the scanners like rpm-verify, yum-update or rpm-update. This will help us differentiating scanners. All the scanners fall under pipeline-scanner, so may be we can change to make it more specific